examples/platform/silabs/SiWx917/SiWx917/sl_wifi_if.c

```diff
@@ -233,14 +221,14 @@ int32_t wfx_rsi_power_save()
  *****************************************************************************************/
 int32_t wfx_wifi_rsi_init(void)
 {
-    sl_status_t status;
-    status = sl_wifi_init(&config, default_wifi_event_handler);
-    if (status != SL_STATUS_OK)
-    {
-        SILABS_LOG("wfx_wifi_rsi_init failed %x", status);
-        return status;
-    }
+  SILABS_LOG("wfx_wifi_rsi_init started");
+  sl_status_t status;
+  status = sl_wifi_init(&config  , default_wifi_event_handler);
+  if(status != SL_STATUS_OK){
+    SILABS_LOG("wfx_wifi_rsi_init failed %x", status);
     return status;
```

Comment:
redundant return with the one at the end of function 2 line down. Also, the return type of the function should be sl_status_t in this case?